src/DotNetty.Common/Utilities/TimeUtil.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Common.Utilities
+{
+    using System;
+
+    /// <summary>
+    /// Time utility class.
+    /// </summary>
+    public static class TimeUtil
```

Comment:
I have changed IdleStateHandler and I am having there two methods, which I think, needs to be moved into a utils class.

```csharp
        TimeSpan Max(TimeSpan t1, TimeSpan t2)
        {
            return TimeSpan.Compare(t1, t2) == 1 ? t1 : t2;
        }

      TimeSpan GetSystemTime()
        {
            return TimeSpan.FromMilliseconds(Environment.TickCount);
        }
```

Where do you think is better place for they?